### ISSUE TITLE
Can't RUN on iOS.  App crashes on require('com.williamrijksen.onesignal');

### ISSUE THREAD

**Issue by chrishaff** [williamrijksen/com.williamrijksen.onesignal#12]
The app crashes immediately when it encounters the line
```javascript
var onesignal = require('com.williamrijksen.onesignal');
```

This happens when I "run" in development mode or package and deploy via TestFlight. The module works fine, receives messages, etc. when I run it in **debug mode**.  Android works fine in any mode.

Ti SDK 5.5.1
Module versions 1.2.1 and 1.3
Latest release versions of Studio, XCode, etc.

Any pointers would be appreciated, thanks!

**Comment by williamrijksen** [comment]
@chrishaff What do you mean with Debug mode? Could you deliver a stacktrace-error?

**Comment by chrishaff** [comment]
I'll post a stack trace. Didn't see anything clearly out of place.

By debug I mean that I compile for debug targeting the physical iPad and
step through the code using Studio. If I hit "run" in Studio the app
crashes when it hits the include line. Same if I deploy through Testflight.

**Comment by chrishaff** [comment]
Sorry, the holidays intervened.  

Here's a gist with the device crash log.
https://gist.github.com/chrishaff/86b3c1dea0bd0176bc8f74368fef578c

I tried putting a try/catch around the require(), but that didn't do a thing as I suspected.  I did notice that while the app starts/stops and leaves you at the springboard I can see the red screen of death when I view running apps.  It says "Can't find onesignal.js".  I tried:

- didn't find such a file "onesignal.js" in this repo.
- I don't have a file "onesignal.js" mentioned in my code anywhere.
- I'm just using the provided example app.js with my regular main project.
- re-installing the plugin to no avail. Tried 1.3.0 and 1.2.1
- disabled all other plugins.

Something going wrong in the build process?  

![screen shot 2016-12-30 at 10 00 16 am](https://cloud.githubusercontent.com/assets/291527/21567057/1c2186ea-ce77-11e6-8644-6d72f990e597.png)

**Comment by aurusapps** [comment]
I had a similar error and I singled it down to a line I used for an android quick fix suggested in #9 
While building for iOS I simply removed this line and it now works in debug and build mode.

**Comment by williamrijksen** [comment]
@chrishaff I'm sorry, I need more information to reproduce you're problem. In our application it works as expected in both Testflight and develop. How do you call the onesignal module in Javascript?

**Comment by chrishaff** [comment]
Thanks @williamrijksen. This morning I created a new app, got certs, etc. and used the example code from this repo.  That worked!

I then went back to the actual app and found some references to Ti.Network.registerForPushNotifications from the previous plugin I used with Parse.  That code is unreachable, that is there is no include() or require() for the file. I had previously removed the plugin.  I moved it out of /resources and things started working.  

Thanks for hanging in there!
